### ISSUE TITLE
SymbolServer.getstore waiting

### ISSUE THREAD

**Issue by zxj5470** [JuliaEditorSupport/SymbolServer.jl#21]
```julia
julia> using LanguageServer
julia> serve=LanguageServerInstance(stdin,stdout,true,"C:\\Users\\zh\\.julia\\environments\\v1.1","",Dict())
Root: Set(String[])

julia> run(serve)
[ Info: Started symbol server

Please submit a bug report with steps to reproduce this fault, and any error messages that follow (in their entirety). Thanks.
Exception: EXCEPTION_ACCESS_VIOLATION at 0x43a2ffc11 -- jl_excstack_state at /home/Administrator/buildbot/worker/package_win64/build/src\rtutils.c:287
in expression starting at C:\Users\zh\.julia\packages\SymbolServer\brsKC\src\clientprocess\clientprocess_main.jl:12
ERROR: InterruptException:
Stacktrace:
 [1] process_events at .\libuv.jl:98 [inlined]
 [2] wait() at .\event.jl:246
 [3] wait(::Condition) at .\event.jl:46
 [4] wait_readnb(::Base.PipeEndpoint, ::Int64) at .\stream.jl:368
 [5] read(::Base.PipeEndpoint, ::Type{UInt8}) at .\stream.jl:822
 [6] read(::Base.Process, ::Type{UInt8}) at .\io.jl:229
 [7] deserialize at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.1\Serialization\src\Serialization.jl:731 [inlined]
 [8] deserialize(::Base.Process) at C:\cygwin\home\Administrator\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.1\Serialization\src\Serialization.jl:718
 [9] request(::SymbolServerProcess, ::Symbol, ::Pair{String,Base.UUID}) at C:\Users\zh\.julia\packages\SymbolServer\brsKC\src\SymbolServer.jl:39
 [10] load_package(::SymbolServerProcess, ::Pair{String,Base.UUID}) at C:\Users\zh\.julia\packages\SymbolServer\brsKC\src\SymbolServer.jl:126
 [11] getstore(::SymbolServerProcess) at C:\Users\zh\.julia\packages\SymbolServer\brsKC\src\SymbolServer.jl:74
 [12] run(::LanguageServerInstance) at C:\Users\zh\.julia\packages\LanguageServer\GNDpW\src\languageserverinstance.jl:34
 [13] top-level scope at none:0
```
![image](https://user-images.githubusercontent.com/20026798/54866627-34174080-4db1-11e9-8d61-bdd523901e1c.png)

version Julia 1.1.0
Windows 10 [10.0.17763.379]
```
(v1.1) pkg> status
  [2b0e0bc5] LanguageServer v0.5.1
  [b3cc710f] StaticLint v0.1.1
  [cf896787] SymbolServer v0.1.2
```

**Comment by tribut** [comment]
Should be fixed by #19 and #22

**Comment by davidanthoff** [comment]
Fantastic!
